src/atoms/link.tsx

```diff
@@ -0,0 +1,22 @@
+import {Link as MuiLink} from "@mui/material"
+
+interface LinkProps {
+    children: string;
+    linkType: 'primary' | 'secondary';
+    underline: 'none' | 'hover' | 'always';
```

Comment:
This should be optional. Also underline default value should be `always`

Reply:
corrected